lib/ansible/modules/windows/win_psrepository.ps1

```diff
@@ -1,68 +1,229 @@
 #!powershell
 
+# Copyright: (c) 2020, Brian Scholer <@briantist>
 # Copyright: (c) 2018, Wojciech Sciesinski <wojciech[at]sciesinski[dot]net>
 # Copyright: (c) 2017, Daniele Lazzari <lazzari@mailup.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 #Requires -Module Ansible.ModuleUtils.Legacy
+#Requires -Module @{ ModuleName = 'PowerShellGet' ; ModuleVersion = '1.6.0' }
 
 # win_psrepository (Windows PowerShell repositories Additions/Removals/Updates)
 
 $params = Parse-Args -arguments $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
 $name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true
-$source = Get-AnsibleParam -obj $params -name "source" -type "str"
+$source_location = Get-AnsibleParam -obj $params -name "source_location" -aliases "source","module_source","module_source_location" -type "str"
```

Comment:
That's a lot of aliases, I would try to just keep `source` as backwards compatibility and make this the `source_location` option. Because the underlying cmdlet is also `SourceLocation` I don't think having the `module_*` ones should matter too much as long as we document the details of this option in the description.

Reply:
It was to allow for consistency in naming for those who want it, but point taken, I don't feel strongly about keeping them.

Reply:
Oh and just after looking at, these options should try and match up with the return values there. When looking at the result for just 1 repo I get

```
ok: [2019] => {
    "changed": false,
    "invocation": {
        "module_args": {
            "name": "*"
        }
    },
    "repositories": [
        {
            "installation_policy": "Untrusted",
            "name": "PSGallery",
            "package_management_provider": "NuGet",
            "provider_options": {},
            "publish_location": "https://www.powershellgallery.com/api/v2/package/",
            "registered": true,
            "script_publish_location": "https://www.powershellgallery.com/api/v2/package/",
            "script_source_location": "https://www.powershellgallery.com/api/v2/items/psscript",
            "source_location": "https://www.powershellgallery.com/api/v2",
            "trusted": false
        }
    ]
}
```

So having `source_location` and `publish_location` as is but keep the prefix for the script ones.